src/core/src/package_managers/ZypperPackageManager.py

```diff
@@ -64,6 +65,11 @@ def invoke_package_manager(self, command):
             self.composite_logger.log('[ERROR] Package manager was invoked using: ' + command)
             self.composite_logger.log_warning(" - Return code from package manager: " + str(code))
             self.composite_logger.log_warning(" - Output from package manager: \n|\t" + "\n|\t".join(out.splitlines()))
+
+            process_lock_tree = self.get_process_tree_from_package_manager_msg(out)
```

Comment:
rename get_process_tree_from_package_manager_**msg** to get_process_tree_from_pid_in_output

rename process_lock_tree to **process_tree**

Variable name should reflect what it stores, or in this case what the function returns. In our example right now, a process is locked, but this is a generic function, merely searching for a pid in the output and returning additional details regarding that process. We don't only look for a locked process message in the output.

Reply:
That makes sense, thanks. Renamed!